### PR TITLE
libcap

### PR DESCRIPTION
**Pull request by scopatz**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by scopatz** [comment]
Ready for review @conda-forge/staged-recipes. This is a linux-only recipe where `make tests` requires sudo

File: recipes/libcap/build.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -ex
+
+export CFLAGS="${CFLAGS} -DXATTR_NAME_CAPS"
+
+make CC=$CC \
+     prefix=$PREFIX \
+     CFLAGS="${CFLAGS}" \
```
Comment (isuruf) [inline recipes/libcap/build.sh]: Set `LDFLAGS` as well?

**Comment by scopatz** [comment]
Added, ready for review again @conda-forge/staged-recipes

File: recipes/libcap/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://mirrors.edge.kernel.org/pub/linux/libs/security/linux-privs/libcap2/libcap-{{ version }}.tar.xz
+  sha256: dac1792d0118bee6aae6ba7fb93ff1602c6a9bda812fd63916eee1435b9c486a
+  patches:
+    - libcap-makefile.patch
```
Comment (isuruf) [inline recipes/libcap/meta.yaml]: Can you explain and a comment what this does?

**Comment by scopatz** [comment]
Ready for review again!

**Pull request merged**